#! /bin/sh /usr/share/dpatch/dpatch-run
## 412786_xmlwf_man_standard_fix.dpatch by Cameron McCormack and
##     Ardo van Rangelrooij <ardo@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: GROFF expects a macro name after a leading point. So remove
## DP:   the linebreak and avoid the warning.
## DP:
## DP: There is no well-formedness constraint in XML
## DP:   that requires the XML declaration.
## DP:
## DP: <URL:http://bugs.debian.org/412786>

@DPATCH@
diff -urNad expat-2.0.1~/doc/xmlwf.sgml expat-2.0.1/doc/xmlwf.sgml
--- expat-2.0.1~/doc/xmlwf.sgml	2003-01-24 19:34:03.000000000 +0100
+++ expat-2.0.1/doc/xmlwf.sgml	2008-05-07 04:06:39.000000000 +0200
@@ -286,8 +286,7 @@
   This gives a fairly accurate idea of the raw speed of Expat itself
   without client overhead.
   <option>-t</option> turns off most of the output options
-  (<option>-d</option>, <option>-m</option>, <option>-c</option>,
-  ...).
+  (<option>-d</option>, <option>-m</option>, <option>-c</option>, ...).
 	   </para>
         </listitem>
       </varlistentry>
@@ -389,11 +388,6 @@
   <refsect1>
     <title>BUGS</title>
 	<para>
-	According to the W3C standard, an XML file without a
-	declaration at the beginning is not considered well-formed.
-	However, <command>&dhpackage;</command> allows this to pass.
-	</para>
-	<para>
 	<command>&dhpackage;</command> returns a 0 - noerr result,
 	even if the file is not well-formed.  There is no good way for
 	a program to use <command>&dhpackage;</command> to quickly
